src/Polar/PolarStore.cpp

```diff
@@ -289,6 +289,16 @@ static constexpr PolarStore::Item internal_polars[] =
   { _T("SZD-54-2 Perkoz (FT 17m)") /* flat tip */,     442, 0, 98, -0.92, 174, -4.35, 250, -13.22, 16.36, 160, 0 },
   { _T("SZD-54-2 Perkoz (WL 17m)") /* winglet */,      442, 0, 99, -0.86, 175, -4.22, 250, -13.01, 16.36, 160, 0 },
   { _T("SZD-54-2 Perkoz (WL 20m)") /* long winglet */, 442, 0, 91, -0.69, 170, -3.98, 250, -12.66, 17.30, 160, 0 },
+  // from Cumulus project
+  { _T("Arcus M (PIL)"), 680, 185, 105, -0.595, 153, -1.1, 197, -2, 15.6, 0, 116 },
+  { _T("Arcus M (PAS)"), 750, 185, 118, -0.669, 159, -1.062, 210, -2, 15.6, 0, 116 },
+  { _T("ASK-18"), 310, 0, 75, -0.613, 138, -1.773, 200, -4.234, 12.99, 0, 88 },
+  { _T("ASW-20BL"), 400, 126, 95, -0.628, 148, -1.338, 200, -2.774, 10.49, 0, 112 },
+  // 180
+  { _T("ASW-24"), 350, 159, 108.82, -0.728, 142.25, -1.213, 167.41, -1.766, 10, 0, 107 },
```

Comment:
@hjr ASW-24 is already part of the list 🤔

Reply:
Thanks for double checking. I removed line 298 with ASW-24.